Sources/ExtensionKit/UIExtensions/UIStackView/UIStackView+ removeAllArrangedSubviews.swift

```diff
@@ -0,0 +1,25 @@
+//
```

Comment:
The filename has an empty state. consider changing from "UIStackView+ removeAllArrangedSubViews" to "UIStackView+RemoveAllArrangedSubViews"